sig-multicluster/namespace-sameness-position-statement.md

```diff
@@ -88,14 +92,42 @@ are running on other clusters in the group.
 As in example 2, cluster-ops run a per-cluster metrics service,  It does not
 make sense for clients in cluster A to access the metrics server of cluster B.
 Even though this runs in the same "metrics" namespace in both clusters (and
-thus namespce sameness applies), the implementation of multi-cluster services
+thus namespace sameness applies), the implementation of multi-cluster services
 probably should not be on-by-default.  The details of how multi-cluster
-services will work are an area for innovation, but ideas include:
-  * Opt-in: services must be "exported" to be merged across clusters
-  * Opt-out: services or namespace can be opted out of service merging
+services work are out of scope for this document, but valid approaches could
+include:
+  * Opt-in: services must be explicitly "exported" to be merged across clusters
+  * Opt-out: services or namespaces can be opted out of service merging
   * Different discovery: merged services and "raw" services use different names
     or other discovery mechanisms
 
-However the implementation works, the metrics service is not automatically
-merged across clusters, though the LDAP-to-RBAC sync from example 2 can still
-apply consistent policies.
+However the implementation works, the metrics service is not _necessarily_
+merged across clusters, even if the LDAP-to-RBAC sync from example 2 still
+applies consistent RBAC policies.  Those are independent capabilities.
+
+### Example 4: Authority sameness policy
```

Comment:
A comment thread got lost on the push:

https://github.com/kubernetes/community/pull/6748#discussion_r1147785678